esp/services/ws_sql/ws_sqlService.cpp

```diff
@@ -922,17 +922,17 @@ bool CwssqlEx::onExecuteSQL(IEspContext &context, IEspExecuteSQLRequest &req, IE
         if (compiledwuid.length() != 0)
            normalizedSQL.append("--PWUID=").append(compiledwuid.str());
 
-        ESPLOG(LogMax, "WsSQL: getWorkUnitFactory...");
+        DBGLOG("WsSQL: getWorkUnitFactory...");
         Owned<IWorkUnitFactory> factory = getWorkUnitFactory(context.querySecManager(), context.queryUser());
 
-        ESPLOG(LogMax, "WsSQL: checking query cache...");
+        DBGLOG("WsSQL: checking query cache...");
         if(cacheeligible && getCachedQuery(normalizedSQL.str(), compiledwuid.s))
         {
-           ESPLOG(LogMax, "WsSQL: cache hit opening wuid %s...", compiledwuid.str());
+            DBGLOG("WsSQL: cache hit opening wuid %s...", compiledwuid.str());
```

Comment:
Nitpicking, but extra leading space.

Reply:
should be fixed here: https://track.hpccsystems.com/browse/HPCC-27246